README.md

```diff
@@ -46,18 +46,23 @@ bundle:dist
 
 ## Typesafe ConductR Bundles
 
-Typesafe ConductR has its own bundle format in order for components to be described. In general there is a one-to-one correlation between a bundle and a component, but it is also possible to have multiple components per bundle. You may want to do this when there is a strong dependency between one component and another. For example, perhaps a Play 2.2 applications requires a specific version of Nginx to proxy it (not that this is a real situation, just an example).
+Typesafe ConductR has a bundle format in order for components to be described. In general there is a one-to-one correlation between a bundle and a component.
 
-Bundles provide Typesafe ConductR with some basic knowledge about components in a *bundle descriptor*; in particular, what is required in order to start a component. The following is an example of a `bundle.conf` descriptor:
+Bundles provide Typesafe ConductR with some basic knowledge about components in a *bundle descriptor*; in particular, what is required in order to load and run a component. The following is an example of a `bundle.conf` descriptor:
 ([Typesafe configuration](https://github.com/typesafehub/config) is used):
 
 ```
 version = "1.0.0"
+system     = "simple-test-0.1.0-SNAPSHOT"
+nrOfCpus   = 1.0
+memory     = 67108864
+diskSpace  = 10485760
+roles      = ["web-server"]
 components = {
   "angular-seed-play-1.0-SNAPSHOT" = {
     description      = "angular-seed-play"
     file-system-type = "universal"
-    start-command    = ["angular-seed-play-1.0-SNAPSHOT/bin/angular-seed-play"]
+    start-command    = ["angular-seed-play-1.0-SNAPSHOT/bin/angular-seed-play", "-Xmm=67108864", "-Xmx=67108864"]
```

Comment:
I am hoping memory settings can be managed by sbt-native-manager when https://github.com/sbt/sbt-native-packager/pull/510 gets merged.